tools/cli/go-whisk-cli/main.go

```diff
@@ -48,6 +48,7 @@ func init() {
 
     // Rest of CLI uses the Properties struct, so set the build time there
     commands.Properties.CLIVersion = CLI_BUILD_TIME
+    os.Setenv("CLI_Version", CLI_BUILD_TIME)
```

Comment:
I was attempting this at first, but isn't as simple. The CLIVersion is intentionally omitted from that config. I assume for good reason?